cognite/client/_api/transformations/schema.py

```diff
@@ -34,17 +42,80 @@ def retrieve(
+                >>> columns = c.transformations.schema.retrieve(destination=TransformationDestination.nodes(view, "InstanceSpace")
+
+
+            Get the schema for a transformation producing edges::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import TransformationDestination
+                >>> from cognite.client.data_classes.transformations.common import ViewInfo
+                >>> c = CogniteClient()
+                >>> view = ViewInfo(space="viewSpace", external_id="viewExternalId", version="viewVersion")
+                >>> columns = c.transformations.schema.retrieve(destination=TransformationDestination.edges(view, "InstanceSpace")
+
+
+            Get the schema for a transformation producing instance type data model::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import TransformationDestination
+                >>> from cognite.client.data_classes.transformations.common import DataModelInfo
+                >>> c = CogniteClient()
+                >>> data_model = DataModelInfo(space="dataModelSpace", external_id="dataModelExternalId",version="dataModelVersion",destination_type="type")
+                >>> columns = c.transformations.schema.retrieve(destination=TransformationDestination.instances(data_model, "InstanceSpace")
+
+
+            Get the schema for a transformation producing instance relationship data model::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import TransformationDestination
+                >>> from cognite.client.data_classes.transformations.common import DataModelInfo
+                >>> c = CogniteClient()
+                >>> data_model = DataModelInfo(space="dataModelSpace", external_id="dataModelExternalId",version="dataModelVersion",destination_type="type", destination_relationship_from_type="relationshipFromType")
+                >>> columns = c.transformations.schema.retrieve(destination=TransformationDestination.instances(data_model, "InstanceSpace")
+
         """
 
-        url_path = utils._auxiliary.interpolate_and_url_encode(self._RESOURCE_PATH + "/{}", str(destination.type))
-        filter = destination.dump(True)
-        filter.pop("type")
-        other_params = {"conflictMode": conflict_mode} if conflict_mode else None
-
-        return self._list(
-            list_cls=TransformationSchemaColumnList,
-            resource_cls=TransformationSchemaColumn,
-            method="GET",
-            resource_path=url_path,
-            filter=other_params,
-        )
+        if destination.type == "nodes" or destination.type == "edges" or destination.type == "instances":
+            url_path = utils._auxiliary.interpolate_and_url_encode(self._RESOURCE_PATH + "/{}", "instances")
+            filter = destination.dump(True)
```

Comment:
unused

Reply:
Added condition to defined value for `url_path` for instances and all the other schemas.  

```
if destination.type in ["nodes", "edges", "instances"]:
            url_path = utils._auxiliary.interpolate_and_url_encode(self._RESOURCE_PATH + "/{}", "instances")
        else:
            url_path = utils._auxiliary.interpolate_and_url_encode(self._RESOURCE_PATH + "/{}", str(destination.type))
```

https://developer.cognite.com/api/v1/#tag/Schema/operation/getFlexibleDataModelInstanceSchema